source/_posts/2019-10-19-hexo-renderer-jade-0-5_hexo-renderer-pug-1.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Official plugins hexo-renderer-jade 0.5.0 & hexo-renderer-pug 1.0.0 released
+---
+
+Since the upstream [jade] package has been replaced by [pug] package and no longer maintained, we've decided to deprecate [hexo-renderer-jade](https://github.com/hexojs/hexo-renderer-jade) and replace it with [hexo-renderer-pug].
```

Comment:
Should we also add link for hexo-rednerer-pug?

Reply:
The link of `hexo-renderer-pug` is already added.

@curbengh is using a markdown syntax called `reference links` and the URL is at L44.

The preview of this PR can be found [here](https://deploy-preview-1185--hexo-site.netlify.com/news/2019/10/19/hexo-renderer-jade-0-5_hexo-renderer-pug-1/).

Reply:
Thank you @SukkaW . The preview is so cool. 😁 

I think we should keep the format consistent. The hexo-renderer-jade is using the direct link, but the hexo-renderer-pug is the reference.